README.md

```diff
@@ -31,8 +33,8 @@ Best place to start is our [contribution guide](/CONTRIBUTING.md).
 
 ----
 
-*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*  
-*Documentation/examples are licensed under [Creative Commons BY-SA 4.0](/docs/LICENSE).*  
+*Code is licensed under the [Apache License, Version 2.0](/LICENSE).*
```

Comment:
The 2 empty spaces are necessary for normal markdown. Maybe that was the issue on Github itself.